R/densityplot.R

```diff
@@ -563,7 +563,7 @@ panel.densityplot.flowFrame <-
           ptList<-plotType("gdensity", parm)
          
           xx <- exprs(frame)[,channel.x.name]
-          r <- unlist(range(frame)[, channel.x.name])
+          r <- unlist(range(frame[, channel.x.name]))
```

Comment:
This should actually be:
`r <- unlist(range(frame[, channel.x.name, drop=FALSE]))`